### PR TITLE
Update CHANGELOG

### PR DESCRIPTION
**Pull request by Honey14**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,11 +1,14 @@
 # CHANGELOG
 
 ## Next Release
+### Internal
```
Comment (msasikanth) [inline CHANGELOG.md:4]: This should be `Feature` right?

Comment (vinaysshenoy) [reply CHANGELOG.md:4]: It should be in "Changes", I think. A feature is something that the user can use, but this is not something that the nurse can take action on.

Comment (Honey14) [reply CHANGELOG.md:4]: Can you tell me the difference between Internal | Changes ?

Comment (vinaysshenoy) [reply CHANGELOG.md:4]: This was my bad, I should have caught this when we paired on this PR.

Internal -> "Not a user facing change"
"Feature" -> "A new tool which the user can use"
"Changes" -> "A user facing change, but which is not in control of the user"
"Fixes" -> "Bug fixes"

**Comment by msasikanth** [comment]
re-rolling the PR on to master branch

**Pull request merged**